sanitize_test.go

```diff
@@ -152,8 +152,8 @@ func TestLinks(t *testing.T) {
 			expected: `<img src="giraffe.gif"/>`,
 		},
 		{
-			in:       `<img src="giraffe.gif?height=500&width=500" />`,
-			expected: `<img src="giraffe.gif?height=500&width=500"/>`,
+			in:       `<img src="giraffe.gif?height=500&width=500&flag" />`,
+			expected: `<img src="giraffe.gif?height=500&width=500&flag"/>`,
```

Comment:
I was thinking about test case closer to what we got reported. Add to `TestLinkTargets` below:

````go
{
  in:       `<a href="https://example.com/?https://example.org/file.html"></a>`,
  expected: `<a href="https://example.com/?https%3A%2F%2Fexample.org%2Ffile.html"></a>`,
},
````

Reply:
this is a better test case because it asserts non-sorting of the query values too and ensures that the flags work even when there are more than just on key/value.

Reply:
Yeah you can keep it, but would still prefer if you add mine too. The more tests the better 😉